tools/database/dbc/spell_effect.go

```diff
@@ -4,12 +4,13 @@ import (
 	"math"
 	"slices"
 
+	"github.com/wowsims/cata/sim/core"
 	"github.com/wowsims/cata/sim/core/proto"
 	"github.com/wowsims/cata/sim/core/stats"
 )
 
 const MAX_SCALING_LEVEL = 100
-const BASE_LEVEL = 85
+const BASE_LEVEL = 90
```

Comment:
Is this maybe why you were getting the wrong stat values for the boosted trinkets?